### PR TITLE
Add Buildasaur v0.6.3

### PR DESCRIPTION
**Pull request by czechboy0**

### CODE REVIEW

File: Casks/Buildasaur.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'Buildasaur' do
```
Comment (vitorgalvao) [inline Casks/Buildasaur.rb]: Needs to be lowercase.

File: Casks/Buildasaur.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version '0.6.3'
+  sha256 '347edecdade78a3959bf6c00f43d0d5b7eb33e78b1f53363b36cc8b61f3593f5'
+
+  url 'https://github.com/czechboy0/Buildasaur/releases/download/v0.6.3/Buildasaur.app.zip'
```
Comment (vitorgalvao) [inline Casks/Buildasaur.rb]: It’s lacking the appcast.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 241b722623c48535527e2d72d3b330b7a8515004. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/Buildasaur.rb (hunk 1 above)
Comment (czechboy0) [inline Casks/Buildasaur.rb]: Ok, also the cask file name? (e.g. `buildasaur.rb`?)

File: Casks/Buildasaur.rb (hunk 2 above)
Comment (czechboy0) [inline Casks/Buildasaur.rb]: Sorry, not sure what that is

**Comment by czechboy0** [comment]
Amazing, thanks @vitorgalvao! I see what you meant by both comments now, thanks again!

**Pull request closed without merging**